nixos/tests/ly.nix

```diff
@@ -0,0 +1,34 @@
+import ./make-test-python.nix ({ lib, ... }:
+
+{
+  name = "ly";
+
```

Comment:
Because of the stuck issue, please add a `globalTimeout` for the test. In my machine it arrives at network phase in about 8s but something around two minutes (120) should be fine if someone is running the test on a hard disk machine.